### PR TITLE
Autogenerate some sip files

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: src/core/qgsactionscope.h [hunk 1]
```diff
@@ -125,6 +125,6 @@ class CORE_EXPORT QgsActionScope
     QgsExpressionContextScope mExpressionContextScope;
 };
 
-CORE_EXPORT uint qHash( const QgsActionScope &key, uint seed = 0 );
+CORE_EXPORT uint qHash( const QgsActionScope &key, uint seed = 0 ); // SIP_SKIP
```
Comment (3nids) [inline src/core/qgsactionscope.h]: you shoud/can use `CORE_EXPORT uint qHash( const QgsActionScope &key, uint seed = 0 ) SIP_SKIP;`

SIP_SKIP was added as a macro too.

File: src/core/qgis.h [hunk 2]
```diff
@@ -447,3 +447,8 @@ typedef unsigned long long qgssize;
   * specify an alternative default value for SIP methods
   */
 #define SIP_PYDEFAULTVALUE(value)
+
+/**
+ * http://pyqt.sourceforge.net/Docs/sip4/annotations.html?highlight=keepreference#function-annotation-ReleaseGIL
+ */
+#define SIP_RELEASEGIL
```
Comment (3nids) [inline src/core/qgis.h]: put it maybe above with the other annotations (PYTYPE and PYDEFAUTVALUES are not)

**Review by 3nids — COMMENTED** [review]
great 👍

File: python/core/qgsactionscope.sip [hunk 3]
```diff
@@ -1,116 +1,95 @@
...
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-/** \ingroup core
- * An action scope defines a "place" for an action to be shown and may add
```
Comment (nyalldawson) [inline python/core/qgsactionscope.sip]: For some reason this particular class docstring hasn't got translated

Comment (3nids) [inline python/core/qgsactionscope.sip]: There shouldn't be any empty lines between comment block and class definition.

File: src/core/qgis.h (hunk 2 above)
Comment (m-kuhn) [inline src/core/qgis.h]: ok, sounds good to me

File: python/core/qgsactionscope.sip (hunk 3 above)
Comment (m-kuhn) [inline python/core/qgsactionscope.sip]: done

**Pull request merged**